src/lib.rs

```diff
@@ -402,8 +393,8 @@ where
 
 impl<T> AtomSetOnce<Box<T>> {
     /// If the Atom is set, get the value
-    pub fn get_mut(&mut self, order: Ordering) -> Option<&mut T> {
-        let ptr = self.inner.inner.load(order);
+    pub fn get_mut(&mut self) -> Option<&mut T> {
+        let ptr = self.inner.inner.load(Ordering::Acquire);
```

Comment:
Getting `&mut Self` would require some synchronization anyway to properly order the previous accesses made through `&Self`, so non-atomic access can be used here.

```suggestion
        let ptr = *self.inner.inner.get_mut();
```